### PR TITLE
JIB-5: add Project structure, AH, PAT and basic GP config

### PR DESCRIPTION
**Pull request by chinzou**
https://mekomsolutions.atlassian.net/projects/JIB/issues/JIB-5

- Create a Project structure
- Add AH configuration and AH entries
- Add Patient Identifier Types
- ADD basic and default locale GPs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  </organization>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>create-distribution</id>
```
Comment (mks-d) [inline pom.xml]: This should be something else. I realise the same string is there for Cambodia. We should move both to something like:
```xml

<id>create-package</id>
```

File: configuration/addresshierarchy/addressConfiguration.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<addressConfiguration>
+  <wipe>true</wipe>
+  <addressComponents>
+    <addressComponent>
+      <field>COUNTRY</field>
+      <nameMapping>Country</nameMapping>
+      <sizeMapping>40</sizeMapping>
+      <elementDefault>addresshierarchy.Bahrain</elementDefault>
```
Comment (mks-d) [inline configuration/addresshierarchy/addressConfiguration.xml]: This suggests that the default element is coded, but
1. For now we won't support i18n.
2. The CSV references `"Bahrain"` and not `"addresshierarchy.Bahrain"`.

Let's change this to 
```xml
<elementDefault>Bahrain</elementDefault>
```

File: pom.xml (hunk 1 above)
Comment (chinzou) [reply pom.xml]: yes, i used the Cambodian config as reference

File: configuration/addresshierarchy/addressConfiguration.xml (hunk 2 above)
Comment (chinzou) [reply configuration/addresshierarchy/addressConfiguration.xml]: ok, i thought the first part is for filename , thanks for the correction :)

**Pull request merged**